### PR TITLE
feat(vaults): make the names of vaults configurable

### PR DESCRIPTION
**Pull request by HackuJacku**
- Fixes the height gui settings not being applied
- Fixes the naming convention of vaults - now configurable
- Updates API version
- Default vault item is now configurable

### CODE REVIEW

File: forge16/src/main/java/com/envyful/vaults/player/PlayerVault.java [hunk 1]
```diff
@@ -36,7 +35,7 @@ public class PlayerVault {
     private ItemStack display;
 
     public PlayerVault(int id) {
-        this(id, "vault#" + (id + 1));
+        this(id, "Vault #" + (id + 1));
```
Comment (danorris709) [inline forge16/src/main/java/com/envyful/vaults/player/PlayerVault.java]: If we're modifying the default then it may as well be in the config

**Review by danorris709 — CHANGES_REQUESTED** [review]

File: forge16/src/main/java/com/envyful/vaults/player/PlayerVault.java (hunk 1 above)
Comment (HackuJacku) [reply forge16/src/main/java/com/envyful/vaults/player/PlayerVault.java]: Sure, will add

File: forge16/src/main/java/com/envyful/vaults/player/PlayerVault.java [hunk 2]
```diff
@@ -36,11 +35,11 @@ public class PlayerVault {
     private ItemStack display;
 
     public PlayerVault(int id) {
-        this(id, "vault#" + (id + 1));
+        this(id, EnvyVaults.getInstance().getConfig().getVaultName().replace("%number%", (id + 1) + ""));
```
Comment (danorris709) [inline forge16/src/main/java/com/envyful/vaults/player/PlayerVault.java:38]: This should be using `String.valueOf`

File: forge16/src/main/resources/META-INF/mods.toml [hunk 3]
```diff
@@ -5,7 +5,7 @@ issueTrackerURL="https://github.com/EnvyWare/EnvyVaults/issues"
 
 [[mods]]
 modId="envyvaults"
-version="1.0.1"
+version="1.0.3"
```
Comment (danorris709) [inline forge16/src/main/resources/META-INF/mods.toml:8]: Not really your place to bump the version of someone else's project

**Review by danorris709 — CHANGES_REQUESTED** [review]

**Comment by danorris709** [comment]
In future each thing should be separated into their own PR

**Comment by danorris709** [comment]
Closing due to inactivity and because I did this myself, properly.

**Pull request closed without merging**